pkgs/misc/flashfocus/default.nix

```diff
@@ -18,6 +18,7 @@ python3Packages.buildPythonApplication rec {
 
   nativeBuildInputs = with python3Packages; [
     setuptools
+    procps
```

Comment:
This won't work, the package will just be available during the build.

Do the same thing as the nc replacement above but with `getExe'` because procps has many executables

Reply:
Right, i was using `buildInputs` locally, but i moved it to `propagatedBuildInputs` where it is put in the path of the program when run. I'd prefer that to patching the python code.

Reply:
Doing patching instead https://github.com/NixOS/nixpkgs/pull/353546